lib/include/prjxray/xilinx/xc7series/crc.h

```diff
@@ -0,0 +1,32 @@
+#ifndef PRJXRAY_LIB_XILINX_XC7SERIES_CRC_H_
+#define PRJXRAY_LIB_XILINX_XC7SERIES_CRC_H_
+
+#include <cstdint>
+
+namespace prjxray {
+namespace xilinx {
+namespace xc7series {
+
+// Extend the current CRC value with one address (5bit) and data (32bit) 
+// pair and return the newly computed CRC value
+
+uint32_t icap_crc(uint32_t addr, uint32_t data, uint32_t prev) {
```

Comment:
I've been using CamelCase for method names.  Please elaborate on how addr and data are provided for a multi-word write.

Reply:
I follow https://google.github.io/styleguide/cppguide.html by habit which says CamelCase for both method and function names unless they are accessors or mutators.  I'm not set in stone on this so I'll leave it up to you to decide whether to change it.